components/camel-slack/src/test/java/org/apache/camel/component/slack/SlackConsumerTest.java

```diff
@@ -18,27 +18,70 @@
 
 import org.apache.camel.builder.RouteBuilder;
 import org.apache.camel.component.mock.MockEndpoint;
+import org.apache.camel.component.slack.helper.SlackMessage;
 import org.apache.camel.test.junit4.CamelTestSupport;
-import org.junit.Ignore;
+import org.apache.http.HttpResponse;
+import org.apache.http.HttpStatus;
+import org.apache.http.client.HttpClient;
+import org.apache.http.client.methods.HttpPost;
+import org.apache.http.entity.StringEntity;
+import org.apache.http.impl.client.HttpClients;
+
+import org.junit.Assert;
+import org.junit.Assume;
+import org.junit.Before;
 import org.junit.Test;
 
-@Ignore
+import org.hamcrest.CoreMatchers;
+
+import java.io.IOException;
+
 public class SlackConsumerTest extends CamelTestSupport {
 
+    private String token;
+    private String hook;
+
+    @Before
+    public void setUp() throws Exception {
+        token = System.getProperty("SLACK_TOKEN");
+        hook = System.getProperty("SLACK_HOOK", "https://hooks.slack.com/services/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");
```

Comment:
Where that hook come from?

Reply:
It is reused from previous version of test. See https://github.com/apache/camel/blob/master/components/camel-slack/src/test/resources/OSGI-INF/blueprint/blueprint.xml#L23